css-pseudo-4/Overview.bs

```diff
@@ -988,6 +988,16 @@ Text and Text Decorations</h4>
     </div>
   </div>
 
+  <a href="https://www.w3.org/TR/css-text-decor/#line-decoration">Line decorations</a> introduced by [=highlight pseudo-elements=]
+  apply only to the text associated with their [=originating element=],
+  and are not propagated to descendants
+  except via property [=inheritance=] (as described [[#highlight-cascade|above]]).
+
+  Note: Unlike the element’s own decorations,
+  decorations declared on the root element’s highlights
```

Comment:
```suggestion
  Note: Unlike the originating element’s own decorations,
  decorations declared on a highlight pseudo-element
```

Reply:
Slight fix, because it's not just the root element's highlight that gets this effect. :)

Reply:
done in 6dbf3d9906f48963aefed24fa7bebd986f9149f6